README.md

```diff
@@ -30,3 +30,8 @@ Conversation [english]
 
 
 End of the project is August 13th.
+
+Launching the application
+
+After downloading the repository, open Windows Powershell and execute the following command after switching in the correct folder.
```

Comment:
They should not download the repository. Instead they will download the JAR from the [releases site](https://github.com/0riginz/C0ffe1sWh8tWeNeed/releases) (once you have a release).